python/pyspark/mllib/fpm.py

```diff
@@ -41,8 +41,11 @@ class FPGrowthModel(JavaModelWrapper):
     >>> model = FPGrowth.train(rdd, 0.6, 2)
     >>> sorted(model.freqItemsets().collect())
     [FreqItemset(items=[u'a'], freq=4), FreqItemset(items=[u'c'], freq=3), ...
+
+    .. addedversion:: 1.4.0
```

Comment:
Should be `versionadded`. Try `make html` under `python/docs`. Please look for compile warnings and check generated html doc.